site/src/components/contributors.astro

```diff
@@ -6,7 +6,8 @@ const CONTRIBUTORS = [];
 const fetchContributors = async (page) => {
   const response = await fetch(`${CONTRIBUTORS_URL}?page=${page}`);
   const contributors = await response.json();
-  CONTRIBUTORS.push(...contributors);
+  if (Array.isArray(contributors))
```

Comment:
The home page crashes locally pretty quickly without this check, as GitHub API has rate limits.